### PR TITLE
Add Filelight

### PR DESCRIPTION
**Pull request by JakobDev**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66331)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by JakobDev** [comment]
It looks like KDE has no stable download links for Windows. Even all existing download Links in the KDE Manifests led to a 403.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I think this is automated via https://github.com/vedantmgoyal2009/vedantmgoyal2009 but vedant's automation is turned off due to automation failures.

You can get the URL to last for a few days by replacing `lastSuccessfulBuild` with the build number from CI.

**Comment by JakobDev** [comment]
I had updated the link to the current working version. Hope it works now.

File: manifests/k/KDE/Filelight/22.04.3/KDE.Filelight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2022-07-07
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://binary-factory.kde.org/job/Filelight_Release_win64/lastSuccessfulBuild/artifact/filelight-22.04.3-1066-windows-msvc2019_64-cl.exe
```
Comment (ItzLevvie) [inline manifests/k/KDE/Filelight/22.04.3/KDE.Filelight.installer.yaml:17]:
```suggestion
  InstallerUrl: https://binary-factory.kde.org/job/Filelight_Release_win64/1066/artifact/filelight-22.04.3-1066-windows-msvc2019_64-cl.exe
```

File: manifests/k/KDE/Filelight/22.04.3/KDE.Filelight.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerSha256: 2853df8622a1cea481ecf4f27ca8716faef7ff8a0a7121c4213bd704d8c1e730
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://binary-factory.kde.org/job/Filelight_Release_win64/lastSuccessfulBuild/artifact/filelight-22.04.3-1066-windows-msvc2019_64-cl.exe
```
Comment (ItzLevvie) [inline manifests/k/KDE/Filelight/22.04.3/KDE.Filelight.installer.yaml:23]:
```suggestion
  InstallerUrl: https://binary-factory.kde.org/job/Filelight_Release_win64/1066/artifact/filelight-22.04.3-1066-windows-msvc2019_64-cl.exe
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: 404 Not Found;

**Pull request closed without merging**